proto/forwarding/forwarding_common.proto

```diff
@@ -168,7 +168,9 @@ enum PacketFieldNum {
   PACKET_FIELD_NUM_ARP_TMAC = 52;             // ARP target mac address.
   PACKET_FIELD_NUM_ARP_SMAC = 53;             // ARP source mac address.
   PACKET_FIELD_NUM_ARP_SPA = 54;              // ARP source protocol address.
-  PACKET_FIELD_NUM_NEXT_HOP_IP = 56; // Next hop ip (packet metadata).
+  PACKET_FIELD_NUM_NEXT_HOP_IP = 56; // Next hop ip (metadata).
+  PACKET_FIELD_NUM_NEXT_HOP_GROUP_ID = 57; // Next hop group id (metadata).
+  PACKET_FIELD_NUM_NEXT_HOP_ID = 58; // Next hop id (metadata).
```

Comment:
What's the difference between nexthop ID and IP? What is the usecase for ID?

Reply:
SAI tracks next hops by an id, it's easier to have that state management in the Lucius